### DISCUSSION TITLE
WordPress not sending email

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#1011] [Q&A] [answered]
I got a report from user that the feedback feature on the listing details page does not work.
She tells me the code she get is this:
"Error: Could not instantiate mail function."

I just tried to use it myself and it does seem to be unresponsive.
I am on version 3.14.8 on Wordpress 6.1, on a Mac. My user is on Windows.
I tested it in Safari and Chrome, neither work.

Feels like a bug? It would be appreciated if this could be corrected asap, as many of my users use this option to notify us about changed.

Thanks!

Nicky
al-anon.alateen.on.ca

**Comment by joshreisner** [comment]
TSML is reporting the error it is getting back from the native `wp_mail()` function, so this sounds like it could be a mail configuration issue on your site, rather than a problem with the plugin. 

This page has a lot of good info on getting mail set up https://kinsta.com/blog/wordpress-not-sending-email/ - The "Check & Log Email" plugin they recommend is a good first step to diagnose the issue.

**Comment by OSAwebsite** [comment]
That seems strange. Someone altogether different from myself it getting it and so do I.
I just disabled the feature in the plugin settings, so it is no longer there, but remind me I have this right:
All I need to do is enter one or more email addresses in that section to activate it, right? Nothing else?
I did not make recent changes.

**Reply by joshreisner** [reply]
I also saw the error when I tried it 15 minutes ago

Yes, adding an address is all you need to do to activate the form. 

But once the form is submitted, the email is handed off to WordPress. If you see an error back, that means WordPress does not think the email was sent. This could be for a variety of reasons, and might not be a change you made. For example, the host might have changed something, perhaps due to an incident with another customer. Or the service you are using, eg Gmail, may have expired credentials, or there could be a temporary sending limit.

If the email is handed off successfully and you don't see an error back, there are also a variety of reasons why the message might not go through. 

At any rate, our support is just limited to the first part.

**Comment by OSAwebsite** [comment]
This is the error showing up. I just reactivated this feature. No idea
where to go from here.


-
Yours in service,
Nicky D
Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*

**Reply by joshreisner** [reply]
Please refer to the link I provided you, you might need to follow up with your hosting company.

**Comment by OSAwebsite** [comment]
OK, thanks for the speedy reply, Josh.


-
Yours in service,
Nicky D
Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*

**Comment by OSAwebsite** [comment]
OK, I scanned that article and it refers mostly to fixes in combination
with certain plugins, none of which I use. So what would be my next step?
I seems a weird phenomenon: I got the last email through that channel
yesterday. Did not do work on the website then. Suggestions?


-
Yours in service,
Nicky D
Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*

**Comment by 97eighty** [comment, marked as answer]
@OSAwebsite, One method you might use to verify the proper mail function is to add a new user to WordPress. WordPress will send out an email in this process. This would verify that the native WordPress mail is functioning correctly. The 12 Step plugin uses this same method to send mail. For security, many hosting companies turn native SMTP access off and suggest using your mail system to send mail. A good plugin for this is https://wordpress.org/plugins/post-smtp/. This plugin will allow you to send mail through your own account and can help in testing. This problem is definitely a WordPress and hosting issue.

**Comment by OSAwebsite** [comment]
Thanks. I will try that.


-
Yours in service,
Nicky D
Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*

**Comment by OSAwebsite** [comment]
I tried but I have to admit it looks like a complicated thing to do. I hope
I did it right and at the end the test email was not sent,. But I am not
sure that was because of something I did wrong in the setup or that
Wordpress is not sending. So who do I go to for support on this issue? Am I
the only one reporting this?

Do I go to my provider? Or do I go to the Wordpress forums?

I could use some help with this.


-
Yours in service,
Nicky D
Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*

**Reply by 97eighty** [reply]
@OSAwebsite I would start with your provider. Ask them, "How can I send mail from WordPress."

**Comment by OSAwebsite** [comment]
Well, I seemed to have been able to fix this with the help of the suggested
plugin.
I asked a more savvy person for help about the configuration and he got it
to work.
If I understand correctly, the plugin now handles all outgoing email
instead of Wordpress.
The free version allows for 100 per day. Since the plugin page is the only
place on the website where people can use a contact form to submit
comments, I think that will do for us.
Thanks for suggesting it.

It does mean that the Wordpress issue remains unhandled, so if someone
finds out why it fails and what to do about it: please let me know.

I have not contacted my provider yet. I might when I have more time.

Thanks to all who helped!


-
Yours in service,
Nicky D
Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*
